command/credits.go

```diff
@@ -0,0 +1,248 @@
+)
+
+var isWindows bool
+
+var thankYou = `
+     _                    _
+    | |                  | |
+_|_ | |     __,   _  _   | |           __
+ |  |/ \   /  |  / |/ |  |/_)   |   | /  \_|   |
+ |_/|   |_/\_/|_/  |  |_/| \_/   \_/|/\__/  \_/|_/
+                                   /|
+                                   \|
+                              _
+                           o | |                           |
+ __   __   _  _  _|_  ,_     | |        _|_  __   ,_    ,  |
+/    /  \_/ |/ |  |  /  |  | |/ \_|   |  |  /  \_/  |  / \_|
+\___/\__/   |  |_/|_/   |_/|_/\_/  \_/|_/|_/\__/    |_/ \/ o
+
+
+`
+
+func init() {
+	rand.Seed(time.Now().UnixNano())
+	isWindows = runtime.GOOS == "windows"
+
+	RootCmd.AddCommand(creditsCmd)
+
+	creditsCmd.Flags().BoolP("static", "s", false, "Print a static version of the credits")
+}
+
+var creditsCmd = &cobra.Command{
+	Use:   "credits [repository]",
+	Short: "View project's credits",
+	Long: `View animated credits for this or another project.
+
+Examples:
+
+  gh credits            # see a credits animation for this project
+  gh credits owner/repo # see a credits animation for owner/repo
+  gh credts -s          # display a static thank you instead of animating
+  gh credits | cat      # just print the contributors, one per line
+`,
+	Args: cobra.MaximumNArgs(1),
+	RunE: credits,
+}
+
+func credits(cmd *cobra.Command, args []string) error {
+	ctx := contextForCommand(cmd)
+	client, err := apiClientForContext(ctx)
+	if err != nil {
+		return err
+	}
+
+	owner := "cli"
+	repo := "cli"
+	if len(args) > 0 {
+		parts := strings.SplitN(args[0], "/", 2)
+		owner = parts[0]
+		repo = parts[1]
+	}
+
+	type Contributor struct {
+		Login string
+	}
+
+	type Result []Contributor
+
+	result := Result{}
+	body := bytes.NewBufferString("")
+	path := fmt.Sprintf("repos/%s/%s/contributors", owner, repo)
+
+	err = client.REST("GET", path, body, &result)
+	if err != nil {
+		return err
+	}
+
+	out := cmd.OutOrStdout()
+	isTTY := false
+	outFile, isFile := out.(*os.File)
+	if isFile {
+		isTTY = utils.IsTerminal(outFile)
+		if isTTY {
+			// FIXME: duplicates colorableOut
+			out = utils.NewColorable(outFile)
+		}
+	}
+
+	static, err := cmd.Flags().GetBool("static")
+	if err != nil {
+		return err
+	}
+
+	static = static || isWindows
+
+	if isTTY && static {
+		fmt.Fprintln(out, "THANK YOU CONTRIBUTORS!!! <3")
+		fmt.Println()
+	}
+
+	logins := []string{}
+	for x, c := range result {
+		if isTTY && !static {
+			logins = append(logins, getColor(x)(c.Login))
+		} else {
+			fmt.Fprintf(out, "%s\n", c.Login)
+		}
+	}
+
+	if !isTTY || static {
+		return nil
+	}
+
+	lines := []string{}
+
+	thankLines := strings.Split(thankYou, "\n")
+	for x, tl := range thankLines {
+		lines = append(lines, getColor(x)(tl))
+	}
+	lines = append(lines, "")
+	lines = append(lines, logins...)
+	lines = append(lines, "( <3 press ctrl-c to quit <3 )")
+
+	termWidth, termHeight, err := terminal.GetSize(int(outFile.Fd()))
```

Comment:
Note that GetSize might not work on cygwin, where we had to use `tput cols` to measure terminal width https://github.com/cli/cli/blob/a7242f4ca3a487432870ed12e5284195e8948433/utils/table_printer.go#L29-L32

Reply:
when we run under cygwin wouldn't we still see a runtime of `windows`?